SPEC.md

```diff
@@ -6015,22 +6039,21 @@ Example input:
 
 ```json
 {
-  "gatk_haplotype_caller.bam": "ftp://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/NA12878/NIST_NA12878_HG001_HiSeq_300x/RMNISTHS_30xdownsample.bam",
-  "gatk_haplotype_caller.reference": {
-    "id":"Homo_sapiens_assembly38",
-    "fasta": "https://storage.googleapis.com/genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta",
-    "index": "https://storage.googleapis.com/genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta.fai",
-    "dict": "https://storage.googleapis.com/genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.dict"
-  },
-  "gatk_haplotype_caller.interval": "chr1:1000000-1010000"
+  "bam": "https://storage.googleapis.com/genomics-public-data/NA12878.chr20.sample.bam",
```

Comment:
This would be another example I'd remove.

Reply:
I think my suggestion is to keep these in 1.1. and 1.2 and make them optional. Then put a new issue and have them removed in 1.3. This will have the least impact on these releases that are already done.